src/python/pants/engine/process.py

```diff
@@ -363,7 +363,7 @@ async def find_binary(request: BinaryPathRequest) -> BinaryPaths:
             command -v $1
         fi
         """
-    )
+    ).lstrip()
```

Comment:
I think you can alternatively fix by using `"""\` at the front, instead of `"""`. I think that was the intent. Saves an extra function call.

Reply:
I'll use the backslash then. (Just pushed plus added a comment to explain it.)